setup.sh

```diff
@@ -1,7 +1,21 @@
 #!/bin/sh
 
-git submodule init && git submodule update
+#setup for redhat
+if [ -f /etc/redhat-release ]; then
+  git submodule init && git submodule update
 
-sudo yum install -y ruby-devel rubygems-devel gcc-c++ curl-devel rubygem-bundler patch zlib-devel redhat-rpm-config openssl
+  sudo yum install -y ruby-devel rubygems-devel gcc-c++ curl-devel rubygem-bundler patch zlib-devel redhat-rpm-config openssl
 
-bundle install
+  bundle install
+# setup for debian 
+elif [ -f /etc/debian_version ]; then
+  git submodule init && git submodule update
```

Comment:
Both this line and the 'bundle install' are the same for both EL and Debian, no? Can be outside the if statement?